tests/utils.py

```diff
@@ -110,8 +110,12 @@ def __init__(self, pytorch_version_tuple):
         self.min_version = pytorch_version_tuple
         if has_pkg_res:
             self.version_too_old = ver(torch.__version__) < ver(".".join(map(str, self.min_version)))
-        else:
+        elif len(self.min_version) == 2:
             self.version_too_old = get_torch_version_tuple() < self.min_version
+        elif len(self.min_version) == 3:
+            self.version_too_old = tuple(int(x) for x in torch.__version__.split(".")[:3]) < self.min_version
```

Comment:
perhaps not working for torch version like `1.7.0a0+7036e91`

Reply:
I will change the new functionality to support only PyTorch 1.6+ instead of 1.5.1+, so these lines will no longer be needed.

Reply:
I have rolled back the changes made to this file as they are no longer needed.